packages/strapi-parts/src/ModalLayout/ModalLayout.stories.mdx

```diff
@@ -16,7 +17,21 @@ import { Button } from '../Button';
 
 This is the doc of the `ModalLayout` component
 
-## ModalLayout
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/ModalLayout)
+
+## Imports
+
+```js
+import { ModalLayout } from '@strapi/parts/ModalLayout';
+import { ModalHeader } from '@strapi/parts/ModalHeader';
+import { ModalFooter } from '@strapi/parts/ModalFooter';
+import { ModalBody } from '@strapi/parts/ModalBody';
+import { TextButton } from '@strapi/parts/Text';
+import { Row } from '@strapi/parts/Row';
+import { Button } from '@strapi/parts/Button';
+```
+
+## Usage
```

Comment:
Can you add a description?

Reply:
I will add some descriptions and examples in another PR